TramsDataApi.Test/Integration/EstablishmentsIntegrationTests.cs

```diff
@@ -32,6 +32,7 @@ public EstablishmentsIntegrationTests(TramsDataApiFactory fixture)
             _legacyDbContext = fixture.Services.GetRequiredService<LegacyTramsDbContext>();
             _randomGenerator = new RandomGenerator();
             _censusDataGateway = new CensusDataGateway();
+            Dispose();
```

Comment:
The changes to this file aren't needed anymore I don't think following https://github.com/DFE-Digital/trams-data-api/pull/280

Reply:
Yep, I didn't mean to check this in. Have removed it.